biogeochem/FatesAllometryMod.F90

```diff
@@ -530,7 +536,7 @@ subroutine carea_allom(dbh,nplant,site_spread,ipft,crowndamage,c_area,inverse)
           call carea_2pwr(dbh,site_spread,d2bl_p2,d2bl_ediff,d2ca_min,d2ca_max, & 
                crowndamage, c_area, do_inverse)
           capped_allom = .false.
-       case(3)
+       case(3,5)
```

Comment:
@XiulinGao , is this case 5 intentional? I don't see any discussion or comments related to a new case 5 leaf allometry.